cmd/lp-colouring/colouring.go

```diff
@@ -21,17 +21,17 @@ const MSB = (1 << 31)
 
 type VertexProperty struct {
 	Colour         uint32
-	coloursIndexed utils.Bitmap
+	coloursIndexed utils.Bitmap // Mark colours that are already used by neighbours
 }
 
 type EdgeProperty struct {
 	graph.TimestampEdge
 }
 
 type Mail struct {
-	NbrScratch []uint32
-	Pos        uint32
-	Colour     uint32 // Multi-purposed (fake union); used as Wait count on existing (if USE_WAIT_COUNT is true)
+	NbrScratch []uint32 // Colours of neighbours
+	Pos        uint32   // Position of the sender in the receiver's OutEdges
```

Comment:
```suggestion
	Pos        uint32   // Unique identifier of the mail sender, that the mail receiver can use to (re)identify the sender. The value is ordinal, and corresponds uniquely to an edge (that has ever existed) that targets the receiver.
```

Note the Pos value is NOT usable to identify a position in the Out-Edges (despite being an undirected graph). 
The value is also only equivalent to a "position in the In-Edges" if there are no Deletions. (The list of in-edges are currently not stored on the receiver side, so this concept is virtual / abstract).

The value is better described as above; let me know if that makes sense, or I can try explaining better. :)